__tests__/connectRoutes.js

```diff
@@ -114,7 +114,13 @@ describe('middleware', () => {
     expect(location.kind).toEqual('redirect')
     expect(location.type).toEqual('THIRD')
     expect(history.entries.length).toEqual(2)
-    expect(location).toMatchSnapshot()
+    expect(location.prev).toEqual({
```

Comment:
this still needs to be a snapshot, not `toEqual`. update the snapshot just for this test so u dont break other ones:

```js
yarn test --testPathPattern regex-for-test-name

# after it works, replace the snapshots with
yarn test --testPathPattern regex-for-test-name -- -u
```